src/components/Header/HeaderRight/HeaderRight.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import './HeaderRight.css';
+//Importing componenents
+import TabOption from './HeaderRightOption/HeaderRightOption';
+import OPTIONS from './Constants/OPTIONS.js'
+
+  function HeaderRight() {
+    return <div className='tab_options'>
+        {OPTIONS.map((option)=>
```

Comment:
Callback function of map: It is getting created everytime. To avoid creation of this function everytime during render: 
1. Create a separate function for this outside.